src/material/autocomplete/autocomplete.ts

```diff
@@ -122,6 +123,14 @@ export abstract class _MatAutocompleteBase
   }
   _isOpen: boolean = false;
 
+  /** Sets the theme color of the panel. @docs-private */
```

Comment:
nit: put jsdoc tag at beginning of the line:
`@docs-private Sets the theme color of the panel.`